### PR TITLE
migrate to warnings ng plugin

### PR DESCRIPTION
**Pull request by tobias-richter**

### CODE REVIEW

File: test/io/wcm/testing/jenkins/pipeline/LibraryIntegrationTestBase.groovy [hunk 1]
```diff
@@ -169,6 +149,11 @@ class LibraryIntegrationTestBase extends BasePipelineTest {
    */
   TeamsNotificationPluginMock teamsNotificationPluginMock
 
+  /**
+   * Mocks the Office365 connector plugin for MS Teams notifications
```
Comment (bellackn) [inline test/io/wcm/testing/jenkins/pipeline/LibraryIntegrationTestBase.groovy]: this comment looks like a copy-paste error

**Review by bellackn — CHANGES_REQUESTED** [review]
see comment, other than that lgtm

File: test/io/wcm/testing/jenkins/pipeline/LibraryIntegrationTestBase.groovy (hunk 1 above)
Comment (tobias-richter) [reply test/io/wcm/testing/jenkins/pipeline/LibraryIntegrationTestBase.groovy]: You are right, i will adjust that

**Review by bellackn — APPROVED** [review]
👍

**Pull request merged**